src/core/Meganav/Meganav.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import Header, { HeaderSessionState } from "../Header";
+import Icon from "../Icon";
+import Flyout from "../Flyout";
+import { menuItemsForHeader } from "./data";
+import { MeganavMobile } from "./MeganavMobile";
+import Notice from "../Notice";
+
+export type MeganavNoticeBannerProps = {
+  props: {
+    title: string;
+    bodyText: string;
+    buttonLink: string;
+    buttonLabel: string;
+    closeBtn: boolean;
+  };
+  config: {
+    cookieId: string;
+    noticeId: string | number;
+    options: {
+      collapse: boolean;
+    };
+  };
+};
+
+export type MeganavProps = {
+  sessionState: HeaderSessionState;
+  searchDataId: string;
```

Comment:
What is searchDataId? (not a leading question - just curious)

Reply:
@jamiehenson this is what Inkeep attaches to, it was added in #515